setup.py

```diff
@@ -0,0 +1,23 @@
+import setuptools
+import os
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="pidtree-bcc",
+    version="0.2",
+    author="Matt Carroll",
+    author_email="mattc@yelp.com",
```

Comment:
Wasn't quite sure what standard practice was here - entirely happy to change!

Reply:
Probably good as this

Reply:
Ack, thanks :)